### DISCUSSION TITLE
What is the correct configuration option for the RPC URL?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [foundry-rs/foundry#1154] [Q&A] [answered]
In the [config README](https://github.com/gakonst/foundry/tree/252d16ba0d947d409002d696d8f6a1a964188339/config), I can see this:

> ```toml
> eth-rpc-url = "https://mainnet.infura.io"
> ```

But in the [Foundry Book](https://book.getfoundry.sh/reference/config.html?highlight=sender#configuration-format), this config option is not even mentioned.

It also seems odd me to that the above option uses `kebab-case` instead of `snake_case` (like all others do).

What is the correct configuration option for the RPC URL?

**Comment by onbjerg** [comment, marked as answer]
The book is somewhat out of date. I think both kebab-case and snake case is supported cc @mattsse
